Readability.js

```diff
@@ -1745,7 +1746,9 @@ Readability.prototype = {
 
     // XXX implement _reduceNodeList?
     this._forEachNode(element.getElementsByTagName("a"), function(linkNode) {
-      linkLength += this._getInnerText(linkNode).length;
+      var href = linkNode.getAttribute("href");
+      var coefficient = href && href.match(this.REGEXPS.hashUrl) ? 0.2 : 1;
```

Comment:
Nit: `this.REGEXPS.hashUrl.test(href)` is faster (and this code can be hot, so it makes sense to use it here).

In fact, should we just use `href.startsWith("#")` here? That accomplishes the same thing, right?

Reply:
the regexp needs to be there because I found many cases where html contains <a href="#"> and the navigation is managed by JS. Those are quite often buttons and not proper hash links.

Reply:
changed it to `this.REGEXPS.hashUrl.test(href)` in latest commit